aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/Translator.java

```diff
@@ -111,35 +113,22 @@ static UntagLogGroupRequest translateToUntagLogGroupRequest(final String logGrou
                 .build();
     }
 
-    static ResourceModel translateForRead(final DescribeLogGroupsResponse response, final ListTagsLogGroupResponse tagsResponse) {
-        final String logGroupName = streamOfOrEmpty(response.logGroups())
-                .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::logGroupName)
-                .filter(Objects::nonNull)
-                .findAny()
-                .orElse(null);
-        final String logGroupArn = streamOfOrEmpty(response.logGroups())
-                .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::arn)
-                .filter(Objects::nonNull)
-                .findAny()
-                .orElse(null);
-        final Integer retentionInDays = streamOfOrEmpty(response.logGroups())
-                .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::retentionInDays)
-                .filter(Objects::nonNull)
-                .findAny()
-                .orElse(null);
-        final String kmsKeyId = streamOfOrEmpty(response.logGroups())
-                .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::kmsKeyId)
-                .filter(Objects::nonNull)
+    static ResourceModel translateForRead(final DescribeLogGroupsResponse response, final ListTagsLogGroupResponse tagsResponse, final String requestLogGroupName) {
```

Comment:
nit: can you change the name of this method to translateForReadResponse and add comments explaining that this is translating for a read *response* not a read request. the naming of the methods in this class are pretty similar and easy to overlook